.gitignore

```diff
@@ -68,3 +68,4 @@ tests/tests_*
 tests/*.log
 tests/*.trs
 test-suite.log
+tags
```

Comment:
is this on purpose?

Reply:
Its ctags file. I'll create a seprate commit for this or move it out of here.

Reply:
Removed :sweat_smile: